MetaMask/metamask-extension
# fix: Aurora RPC deprecation message

## **Description**

Fixes the Aurora deprecation message.  It was suggesting MM no longer supports Aurora, which is not true.  It's just no longer a built-in network due to Infura no longer supporting it.

Instead, we show a warning to users who previous added it when it was a built-in network.  And give them a 1 click migration from Infura to the `mainnet.aurora.dev` RPC endpoint.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24534?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/23078

## **Manual testing steps**

1. Add the Aurora network via https://chainid.network
2. Verify no deprecation warnings

The RPC migration modal is harder to test.  You'd need to have added the network on MM <= 11.6 back when the Infura RPC url was alive.  

## **Screenshots/Recordings**



### **Before**

<img width="352" alt="Screenshot 2024-05-14 at 10 07 12 PM" src="https://github.com/MetaMask/metamask-extension/assets/3500406/dbabcf56-6f0d-4bb3-95a9-2fb9c04359c4">


### **After**

Simulating a user who previously added the Infura RPC:

https://github.com/MetaMask/metamask-extension/assets/3500406/6df7591d-284f-47bc-88a3-362174bf141f

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
